docs/data-sources/tenants.md

```diff
@@ -18,31 +18,28 @@ Provides information about existing tenants.
 ### Optional
 
 - `cloned_from_tenant_id` (String) A filter to search for a cloned tenant by its ID.
+- `id` (String) The unique ID for this resource.
```

Comment:
This should be read-only as it's an ID set by us

Reply:
Resolved these issues and the same problem across most other data sources, except where blocks are used and I cannot set the Computed attribute.
![image](https://github.com/user-attachments/assets/83765a52-ba7f-49a6-a297-19cf7c825df8)